centrifuge-js/src/CentrifugeBase.ts

```diff
@@ -115,6 +121,16 @@ const parachainRpcMethods: Record<string, Record<string, DefinitionRpc>> = {
       ],
       type: 'Vec<u128>',
     },
+    nav: {
+      description: 'Retrieve the net asset value of a pool',
+      params: [
+        {
+          name: 'pool_id',
+          type: 'u64',
+        },
+      ],
+      type: 'PoolNav',
```

Comment:
Needs to be wrapped into Option
```suggestion
      type: 'Option<PoolNav>',
```